src/commands/android/index.ts

```diff
@@ -420,6 +420,9 @@ export class AndroidSetup {
 
   async getSetupConfigs(options: Options): Promise<SetupConfigs> {
     const configs = this.getConfigFromOptions(options);
+    if (this.options.standalone && !this.options.browsers) {
+      configs.browsers = 'none';
+    }
```

Comment:
This should go inside the `this.getConfigFromOptions` method, after `configs.browsers` is set.

Reply:
You should also add a test for this part (to make sure that `configs.browsers` is set correctly. The test will go in `unit_tests/commands/android/testIndex.js` inside `test getConfigFromOptions` describe block.